### PR TITLE
Performance issue fix for manged sni

### PR DESCRIPTION
**Pull request by geleems**
This is a fix for the issue: #24480 -- .NET Core performance issue on Linux.
This fix was merged to master branch: https://github.com/dotnet/corefx/pull/25621

### CODE REVIEW

**Comment by saurabh500** [comment]
This change looks good. Please make the version changes in this PR as well.

File: src/System.Data.SqlClient/dir.props [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <Import Project="..\dir.props" />
   <PropertyGroup>
     <PackageVersion>4.4.2</PackageVersion>
```
Comment (weshaggard) [inline src/System.Data.SqlClient/dir.props]: Package version needs to be bumped to 4.4.3

**Comment by weshaggard** [comment]
You need to add the package packages.builds file to ensure it gets built. Similar to https://github.com/dotnet/corefx/commit/6bbfaf97365af7c5d76732a8458f674b525f176b#diff-cb73a137838f73872d476fa5146bfede.

This change also needs shiproom approval.

File: pkg/Microsoft.Private.PackageBaseline/packageIndex.json [hunk 2]
```diff
@@ -1146,7 +1146,8 @@
         "4.1.1.0": "4.3.0",
         "4.1.1.1": "4.4.1",
         "4.2.0.0": "4.4.0",
-        "4.2.0.1": "4.4.1"
+        "4.2.0.1": "4.4.1",
+        "4.2.0.2": "4.4.2",
```
Comment (geleems) [inline pkg/Microsoft.Private.PackageBaseline/packageIndex.json]: @weshaggard 
Should this be also "4.2.0.2": "4.4.***3***" ??

Comment (weshaggard) [reply pkg/Microsoft.Private.PackageBaseline/packageIndex.json]: Yes. If you run the `/t:UpdatePackageIndex` from the pkg directory it should update this correctly.

Comment (geleems) [reply pkg/Microsoft.Private.PackageBaseline/packageIndex.json]: Fixed to `"4.2.0.2": "4.4.3"`

File: src/System.Data.SqlClient/dir.props (hunk 1 above)
Comment (geleems) [reply src/System.Data.SqlClient/dir.props]: Fixed to `4.4.3`

**Comment by geleems** [comment]
@weshaggard I also sent shiproom approval request email, and waiting for the response. Can you hit approve if nothing else needs to be changed?

**Pull request merged**